hack/make/build-deb

```diff
@@ -3,6 +3,8 @@ set -e
 
 # subshell so that we can export PATH and TZ without breaking other things
 (
+	# Keep list uptodate with what uses systemd >= 226 until everything does
+	export new_systemd=("ubuntu-xenial" "debian-stretch")
```

Comment:
Since these and everything after them will support it, would it be better to swap this list and list the ones that _don't_ have the newer systemd? (ie, only update this list when we _remove_ unsupported distros, not for every new one we add?)